examples/Web/Controllers/HomeController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Confluent.Kafka;
+
+
+namespace Web.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly Random random;
+        private string topic;
+        private readonly KafkaDependentProducer<Null, string> producer;
+
+        public HomeController(KafkaDependentProducer<Null, string> producer, IConfiguration config)
+        {
+            // In a real-world application, you might be using Kafka as a system of record and may wish
+            // to update application state in your request handlers. Or you may wish to write some
+            // analytics data related to your handler logic. In this example, we aren't doing anything
+            // interesting, so we just write frivolous messages to demonstrate how to set things up.
+            this.topic = config.GetValue<string>("Kafka:FrivolousTopic");
+            this.producer = producer;
+            this.random = new Random();

```

Comment:
`Random` is not threadsafe so if 2 runs use same instance you need to `lock` on `random`

Reply:
yep, good points - thanks. what I will do is move it into the handler and seed it with the current time. we don't care about performance since the whole point of this is to artificially slow things down...